Makefile

```diff
@@ -46,7 +46,10 @@ CRD_APIS :=./pkg/apis/cloudcredential/v1
 # Exclude e2e tests from unit testing
 GO_TEST_PACKAGES :=./pkg/... ./cmd/...
 
-IMAGE_REGISTRY :=registry.ci.openshift.org
+IMAGE_REGISTRY ?=registry.ci.openshift.org
+IMAGE_REPO ?=ocp/4.5
```

Comment:
I get that this is just a refactor... but really?

Reply:
This is what it was hardcoded as but yeah I agree. I needed variables I could manipulate to be able to do something like: `IMAGE_REGISTRY="quay.io" IMAGE_REPO="abutcher/cco" IMAGE_TAG="latest" make images`.

Reply:
Would probably be better to just have one variable for the whole `IMG` like the Hive `Makefile`.

Reply:
Will also need a FUP for local dev documentation updating.